src/index.html

```diff
@@ -11,7 +11,147 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post" onsubmit="onSubmit()"
+    >
+      <fieldset class="inputs-block">
+        <legend>Personal information</legend>
+        <div class="input-form">
+          <label for="surname"> Surname: </label>
+          <input
+            type="text"
+            name="Surname"
+            id="surname" required
+          >
+        </div>
+        <div class="input-form">
+          <label for="name"> Name:</label>
+          <input
+            type="text"
+            name="Name"
+            id="name" required
+          >
+        </div>
```

Comment:
переносить нужно каждый атрибут
```suggestion
          <label for="name"> Name:</label>
          <input
            type="text"
            name="Name"
            id="name"
            required
          >
        </div>
```